### ISSUE TITLE
Managing password in environment variable

### ISSUE THREAD

**Issue by markand** [MusicPlayerDaemon/ncmpc#16]
Hello,

Since the #mpd channel is censored by a dictator asshole (excerpt verbatim from the conversation)

    20:43 -!- Irssi: Join to #mpd was synced in 1 secs
    20:43 < markand> hello there
    20:44 < markand> is there something wrong with MPD_HOST environment variable for ncmpc? I've tried export MPD_HOST=the_password@the_hostname
    20:44 < markand> ncmpc connects fine but it does not use the password, any step requires to type it
    20:44 < markand> and more importantly, it's written at the top “Queue on the_password@the_hostname”
    20:45 <@cirrus> how do you know it does not use the password?
    20:45 < markand> because if I try any command it prompts me the password
    20:46 < markand> and being printed on the top of the screen I thought it was ignored
    20:46 <@cirrus> so you did not actually verify your assertion?
    20:47 < markand> cirrus, I have not verified the source code, I just wondering why I should have to pass the password both in MPD_HOST and each time I need to do an operation
    20:48 <@cirrus> I did not ask whether you have verified the source code.
    20:49 <@cirrus> at this point, it's time for you to start reading the channel topic.
    20:49 < markand> cirrus, what is it always that hard to speak with you? it's not the first time you have a very condescending talk with me
    20:50 < markand> this is not encouraging any user contribution
    20:50 <@cirrus> ok, I'll stop replying to you.
    20:50  * Rasi understood the question just fine
    20:51 < markand> yeah, ten years old reaction
    20:51 -!- markand was kicked from #mpd by cirrus [bye]
    20:51 -!- markand [~markand@unaffiliated/markand] has joined #mpd
    20:51 -!- Topic for #mpd: Music Player Daemon | Latest 0.20.19 | http://www.musicpd.org/ | Problems? Just ask away, and please provide the info as explained on: http://www.musicpd.org/help/
    20:51 -!- Topic set by cirrus [] [Thu Apr 26 20:00:59 2018]
    20:51 [Users #mpd]
    20:51 [@cirrus      ] [ Civil        ] [ frispete   ] [ lagothrix  ] [ pyllyukko      ] [ VargaD     ]
    20:51 [@Rasi        ] [ codebam      ] [ frobnicator] [ lvlinux    ] [ ravndal        ] [ veremitz   ]
    20:51 [ _dev_zero   ] [ Commander1024] [ gpol       ] [ lynxis     ] [ SCHAPiE        ] [ winny      ]
    20:51 [ ajashton    ] [ craigger     ] [ hl         ] [ markand    ] [ sentriz        ] [ wolfshappen]
    20:51 [ Alam_Squeeze] [ divx118      ] [ Hobbyboy   ] [ Mindi      ] [ soapsurfer     ] [ x73        ]
    20:51 [ ArsenArsen  ] [ djselbeck    ] [ HumanG33k  ] [ minimalism ] [ souther        ] [ xcm        ]
    20:51 [ ashka       ] [ DogBoy       ] [ i7c        ] [ moviuro    ] [ sqozz          ] [ xSmurf     ]
    20:51 [ at0m        ] [ doppo        ] [ infinisil  ] [ NightMonkey] [ storrgie       ] [ zeltak     ]
    20:51 [ atomi       ] [ dysfigured   ] [ jnes       ] [ npanic     ] [ SuperTux88     ] [ zyliwax    ]
    20:51 [ belak       ] [ emilsp       ] [ KannOx     ] [ ntnn       ] [ swimmer        ]
    20:51 [ brethil     ] [ eu           ] [ katie_     ] [ oldlaptop  ] [ sydney_untangle]
    20:51 [ buZz        ] [ felixsanz    ] [ ketas      ] [ poxifide   ] [ szgyg          ]
    20:51 [ catern      ] [ FichteFoll   ] [ keur       ] [ prg318     ] [ thesquib       ]
    20:51 [ chowbok     ] [ friedrich_   ] [ KittyKatt  ] [ Psy-Q      ] [ Valfor         ]
    20:51 -!- Irssi: #mpd: Total of 79 nicks [2 ops, 0 halfops, 0 voices, 77 normal]
    20:51 -!- Channel #mpd created Sun Nov 26 07:42:55 2006
    20:51 -!- Irssi: Join to #mpd was synced in 1 secs
    20:51 < markand> I didn't know it's so hard to understand someone asking why password is required twice
    20:51 -!- mode/#mpd [+b *!*markand@unaffiliated/markand] by cirrus
    20:51 -!- markand was kicked from #mpd by cirrus [I SAID BYE]

Basically, I'm wondering why is the password required twice and printed on the top of the screen, example:

    export MPD_HOST=password@hostname
    export MPD_PORT=35000

Now, if I run `ncmpc`, on the top of the screen the password will be printed in clear “Queue on password@hostname” and each command require to type the password again. Should it be better to keep it and avoid printing it on the screen?

**Comment by MaxKellermann** [comment]
The "dictator asshole" says: you're banned from this whole project!
